wiki/Game_modifier/Free_mod/en.md

```diff
@@ -0,0 +1,26 @@
+---
+stub: true
+tags:
+  - free mod
+  - free-mod
+  - freemod
+  - FM
+  - tournament category
+  - tournament
+---
+
+# Free mod (tournament category)
+
+*For the full list of all mods, see: [Game Modifier](/wiki/Game_modifier)*\
+*See also: [No mod](/wiki/Game_modifier/No_mod)*
+
+## About
+
+- Abbreviation: FM
+- Alternative spellings: `Free-mod`, `Freemod`
+
+## Description
+
+**Free mod** is a tournament category used to describe plays where competing teams may use whatever combination of [game modifiers (mods)](/wiki/Game_modifier) that they wish (exlcuding unranked mods).<!--more information needed-->
+
+"Free mod" may also be used within organized [Multiplayer](<!---->) parties/matches to describe similar circumstances: (players/teams playing the selected [beatmap](/wiki/Beatmaps) with whatever mods they wish)
```

Comment:
It looks like this paragraph was not finished...?

Reply:
I think I was too lazy to go and find the link for `Multiplayer` and just forgot abt this. Woops